app/components/Layout/DefaultLayout.tsx

```diff
@@ -51,11 +51,11 @@ const DefaultLayout: React.FC<Props> = ({
             display: flex;
             flex-direction: column;
           }
-
           main {
             padding: 30px 40px;
             flex-grow: 1;
             background-color: #fafafc;
+            margin: auto;
```

Comment:
- The `background-color` rule may need to be moved to the `#page-wrap` rule set, as the side margins are white
- The "Contact Form" page ends up being too narrow, see screenshot below. Maybe a margin to the right of the "New Contact"/"Edit Contact" heading will fix it? 
![image](https://user-images.githubusercontent.com/2684022/156209737-a5ce60ed-158c-4f16-bd90-47973fe5a63b.png)